Production_scripts/A05 Prepare Social Care Client lookup.R

```diff
@@ -0,0 +1,199 @@
+#####################################################
+# Social Care Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Social Care Client data from Platform
+# Description - Get the client extract from DVPROD / social_care_2
+#####################################################
+
+# Load packages
+library(dplyr)
+library(dbplyr)
+library(tidyr)
+library(createslf)
+
+
+# Read in data---------------------------------------
+
+# specify latest year
+latest_year <- 1920
+
+# set-up conection to platform
+db_connection <- phs_db_connection(dsn = "DVPROD")
+
+
+# read in data - social care 2 client
+client_data <- tbl(db_connection, in_schema("social_care_2", "client")) %>%
+  select(
+    sending_location, social_care_id, financial_year, financial_quarter,
+    dementia, mental_health_problems, learning_disability,
+    physical_and_sensory_disability, drugs, alcohol, palliative_care,
+    carer, elderly_frail, neurological_condition, autism,
+    other_vulnerable_groups, living_alone, support_from_unpaid_carer,
+    social_worker, type_of_housing, meals, day_care
+  ) %>%
+  collect() %>%
+  filter(financial_year == convert_fyyear_to_year(latest_year)) %>%
+  arrange(
+    sending_location, social_care_id, financial_year,
+    financial_quarter
+  )
```

Comment:
```suggestion
# read in data - social care 2 client
client_data <- tbl(db_connection, in_schema("social_care_2", "client")) %>%
  select(
    sending_location, social_care_id, financial_year, financial_quarter,
    dementia, mental_health_problems, learning_disability,
    physical_and_sensory_disability, drugs, alcohol, palliative_care,
    carer, elderly_frail, neurological_condition, autism,
    other_vulnerable_groups, living_alone, support_from_unpaid_carer,
    social_worker, type_of_housing, meals, day_care
  ) %>%
  filter(financial_year == fy) %>%
  arrange(
    sending_location, social_care_id, financial_year,
    financial_quarter
  )  %>% 
  collect()
```

Could you please test this works? Try put FY as an object e.g. ALT FY. The function convert year types was preventing the SQL from working. But this would reduce the read in time if the filter was done before the collect. Thanks